doc/Programming-guide/Using-Kaa-endpoint-SDKs/index.md

```diff
@@ -6,1107 +6,19 @@ sort_idx: 40
 ---
 Using Kaa endpoint SDKs
 
+{% include variables.md %}
 
-* [Introduction](#introduction)
+* TOC
+{:toc}
 
-## Introduction
+Developing an IoT solution we all the time facing the same routine - creating network communication stack, log delivery functionality, event exchange between endpoints and etcetera.
+All of this functionality already provides the Kaa platform and you can get it out of the box using Kaa endpoint SDK.
```

Comment:
> ...provides the Kaa platform and you can get it out of the box using Kaa endpoint SDK.

here `it` substitutes `Kaa platform`. As far as I understand it should serve as a substitution of `functionality`, isn't it?